### ISSUE TITLE
Is it possible to add meta-l1calo to PetaLinux 2020.2?

### ISSUE THREAD

**Issue by rodrirq** [kratsg/meta-l1calo#26]
Hello, I want to add this layer to my PetaLinux project because I need to get python3-construct. I tried to clone the layer and add it to my petalinux-config, but I am getting errors from meta-xilinx and I don't know why.

**Comment by rodrirq** [comment]
The error is about the device-tree.bbapend. The line SRC_URI_append = " file://${SYSTEM_USER_DTSI} is the reason of the error. But I don't know if this {SYSTEM_USER_DTSI} has been defined somewhere or if I have to change it for the correct dtsi, but I don't know which dtsi I should use.

**Comment by kratsg** [comment]
./cc @easmith5

**Comment by easmith5** [comment]
Hi, the dtsi is defined in the bbappend [right above the line you mention](https://github.com/kratsg/meta-l1calo/blob/master/recipes-bsp/device-tree/device-tree.bbappend#L4-L5). You'll need a device tree file appropriate for your hardware. The lines I linked above are for my specific machines that I've defined in the layer. 

I'm not sure how else to help since I don't know what error you're getting or what your setup is. 

I build with yocto but I'm inside the meta-xilinx v2020.2 setup, so I'd expect it to be compatible with petalinux, but I can't guarantee it.
